README.md

```diff
@@ -128,6 +126,13 @@ Set this variable to deploy partner pods in a custom namespace instead of the de
 export TNF_PARTNER_NAMESPACE="CNF-ns"
 ```
 
+### Specify the image id to be used with oc debug commands
+In disconnected environment, only specific versions of images are mirrored to the local repo. For the `oc debug` command (used by a number of tests) to work, set TNF_OC_DEBUG_IMAGE_ID:
+
+```shell-script
+export TNF_OC_DEBUG_IMAGE_ID="quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:0f5ce898fbad3671fecd6f797130f950fb1abdbaf0d8154c22e2b59c74e3a918"
+```
```

Comment:
is this the one based on rhel-8 ?  Ramon suggested we use that version. I can't check since I don't have access to this repository.

Reply:
not sure. this is just an example. the operator of the disconnected OCP knows what's available.